src/kubernetes/test/local.yaml

```diff
@@ -0,0 +1,71 @@
+
+---
```

Comment:
you running your app with this?
do you think this should be kept around?
interested in your thoughts

Reply:
i only used initially because i had some issues just getting application to start. it turned out it was something else but i left it in case someone else might be keen to explore?

i'm keen on your thoughts :) ive added READMEs for example where I just felt it was lacking something but its just a habit I carried over - anything that could be of use then keep in repo. my last role had zero info in the repos and i did hate it ;)